fem/bilininteg.hpp

```diff
@@ -2089,7 +2089,6 @@ class MixedVectorWeakDivergenceIntegrator : public MixedVectorIntegrator
    }
 };
 
-
```

Comment:
I know we don't specify this explicitly as a rule, however, I think it is better to have two empty lines between classes (in headers) and also between implementations of different classes (in source `.cpp` files). At least let's not change the spacing when it is there. 😁